src/TwbsHelper/Form/View/Helper/FormRow.php

```diff
@@ -105,7 +105,7 @@ public function render(\Laminas\Form\ElementInterface $oElement, $sLabelPosition
      */
     public function renderFormRow(\Laminas\Form\ElementInterface $oElement, $sElementContent): string
     {
-        $aRowClasses = ['form-group'];
+        $aRowClasses = ['mb-3'];
```

Comment:
[Spacing](https://getbootstrap.com/docs/5.0/utilities/spacing/) should not be hardcoded.

Reply:
This spacing is according the official documentation because without this hardcoded spacing we'll need to add the additional class to every element in form factory. In V4 this spacing is added vie form-group class but in V5 this class is removed and all controls renders to close to each other.
Maybe we can add an option to config array to define the default class for spacing?

Reply:
> This spacing is according the official documentation…

This is only an example in the documentation and does not mean that every form must use the same spacing.

> Maybe we can add an option to config array to define the default class for spacing?

I think this is the correct direction.

Reply:
> I think this is the correct direction.

I made this change in new commit.